styx-scheduler-service/src/main/java/com/spotify/styx/state/handlers/FlyteRunnerHandler.java

```diff
@@ -110,7 +112,7 @@ private void pollingExecution(RunState state, EventRouter eventRouter) {
     final String execName = styxExecIdToFlyteNameMapper.apply(executionId);
 
     try {
-      flyteRunner.poll(FlyteExecutionId.create(flyteExecConf.referenceId().project(), flyteExecConf.referenceId().domain(), execName), state);
+      flyteRunner.poll(getExecutionId(state), state);
```

Comment:
is the flyteExecCOnf used anywhere after this refactor? Can it be removed? What about the executionId and execName.
It looks to me that they are not

Reply:
Yes, good catch. That was the reason of introducing the method to avoid repeating those lines